### PR TITLE
update to 1.3.1

### PR DESCRIPTION
**Pull request by dam0vm3nt**
need to add some tests

### CODE REVIEW

File: lib/src/template/dom_repeat.dart [hunk 1]
```diff
@@ -85,6 +85,36 @@ class DomRepeat extends TemplateElement
     jsElement.callMethod('set', ['delay', newVal]);
   }
 
+  /// Defines an initial count of template instances to render after setting
+  /// the `items` array, before the next paint, and puts the `dom-repeat`
+  /// into "chunking mode".  The remaining items will be created and rendered
+  /// incrementally at each animation frame therof until all instances have
+  /// been rendered.
+  num get initialCount => jsElement['initialCount'];
+  void set initialCount(num newVal) {
```
Comment (jakemac53) [inline lib/src/template/dom_repeat.dart]: looks like this could probably be `int`

**Comment by jakemac53** [comment]
looks good once we have some tests, and should also update the pubspec/changelog (1.0.0-rc.16)

**Comment by jakemac53** [comment]
Also we are going to need to update the dependency on `polymer_interop` to `1.0.0-rc.9`.

**Comment by donny-dont** [comment]
1.4 dropped yesterday

**Comment by dam0vm3nt** [comment]
Yeah, fantastic. I will kick that damn beast in before it will become 1.4.1

Il sab 19 mar 2016, 17:02 Don <notifications@github.com> ha scritto:

**Pull request merged**